### PR TITLE
#1399 Web We Want: Welcome new users to the community

### PR DESCRIPTION
**Pull request by jranks123**
New design implemented. Haven't done the name part for now.

See screenshot:

screenshot at apr 07 09-40-59

### CODE REVIEW

File: app/com/gu/identity/frontend/models/text/RegisterConfirmationText.scala [hunk 1]
```diff
@@ -4,24 +4,36 @@ import play.api.i18n.Messages
...
+    weValueYourOpinionP1: String,
+    weValueYourOpinionP2: String,
+    weValueYourOpinionP3: String,
+    completeRegistration: String)
 
 object RegisterConfirmationText {
   def apply()(implicit messages: Messages): RegisterConfirmationText =
     RegisterConfirmationText(
       pageTitle = messages("registerConfirmation.pageTitle"),
+      userName = messages("registerConfirmation.userName"),
```
Comment (nlindblad) [inline app/com/gu/identity/frontend/models/text/RegisterConfirmationText.scala]: Should this be removed?

File: app/com/gu/identity/frontend/models/text/RegisterConfirmationText.scala [hunk 2]
```diff
@@ -4,24 +4,36 @@ import play.api.i18n.Messages
 
 case class RegisterConfirmationText private(
     pageTitle: String,
+    userName: String,
```
Comment (nlindblad) [inline app/com/gu/identity/frontend/models/text/RegisterConfirmationText.scala]: Should this be removed?

File: public/_layout.css [hunk 3]
```diff
@@ -108,6 +108,16 @@ body > section:last-of-type {
   }
 }
 
+/**
+ * Smaller headline
+ */
+.page-header-small {
```
Comment (nlindblad) [inline public/_layout.css]: Maybe rename to `.page-heading--small` to match the previous naming?

File: public/components/register-confirmation/_register-confirmation.css [hunk 4]
```diff
@@ -19,6 +21,17 @@
   text-align: center;
 }
 
+.register-confirm__section {
+  border-top: 2px solid #f4f4f4;
+}
+
+
+.page-content-confirm {
```
Comment (nlindblad) [inline public/components/register-confirmation/_register-confirmation.css]: Maybe change to `--confirm` for BEM naming?

File: public/register-confirmation-page.hbs [hunk 5]
```diff
@@ -2,23 +2,35 @@
...
-        <li class="register-confirm__nav-bar-item"><a class="register-confirm__nav-bar-item-link" href="{{resetPasswordUrl}}">{{registerConfirmationPageText.resetPassword}}</a></li>
-        <li class="register-confirm__nav-bar-item"><a class="register-confirm__nav-bar-item-link" href="{{signOutUrl}}">{{registerConfirmationPageText.signOut}}</a></li>
-    </ul>
+  <section class = "register-confirm__section">
+      <h2 class="page-header-small">{{ registerConfirmationPageText.weValueYourOpinionTitle }}</h2>
+      <section class="page-content">
+        <p class = "page-content-confirm"> {{ registerConfirmationPageText.weValueYourOpinionP1 }} </p>
+        <p class = "page-content-confirm">{{ registerConfirmationPageText.weValueYourOpinionP2}}</p>
+        <p class = "page-content-confirm">{{ registerConfirmationPageText.weValueYourOpinionP3}}</p>
+      </section>
```
Comment (nlindblad) [inline public/register-confirmation-page.hbs]: Fix indentation.

File: public/register-confirmation-page.hbs [hunk 6]
```diff
@@ -2,23 +2,35 @@
...
+
+    <section class="page-content">
+      <a class="register-confirm__complete-registration-button" href="{{returnUrl}}">
+        {{registerConfirmationPageText.completeRegistration}}
+      </a>
+    </section>
+
 {{/partial}}
 {{> layout}}
+
```
Comment (nlindblad) [inline public/register-confirmation-page.hbs]: Maybe remove this newline?

File: public/register-confirmation-page.hbs (hunk 5 above)
Comment (jranks123) [inline public/register-confirmation-page.hbs]: Turns out this wasn't neccassery

**Comment by nlindblad** [comment]
👍

**Pull request merged**